cloudinit/util.py

```diff
@@ -1802,6 +1801,24 @@ def time_rfc2822():
         ts = "??"
     return ts
 
+def boottime():
+    import ctypes
```

Comment:
and move it *into* the `boottime()` function, because no where else is this import used!